<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterBeforeBox</class>
 <widget class="QWidget" name="FilterBeforeBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QCheckBox" name="filterBgShow">
         <property name="text">
          <string>show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="filterBgUpdate">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="filterBgReset">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterBgLoad">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterBgSave">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>36</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QCheckBox" name="filterBgDeleteTrj">
         <property name="toolTip">
          <string>delete trajectory after a certain number of trackpoints outside foreground</string>
         </property>
         <property name="text">
          <string>delete</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_65">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>trj. after:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PSpinBox" name="filterBgDeleteNumber">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_18">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="filterBorder">
     <property name="text">
      <string>border [2px]:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="PSlider" name="filterBorderParamSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PSpinBox" name="filterBorderParamSize_spin">
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterBorderParamCol">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="filterBg">
     <property name="text">
      <string>bg subtr.:</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="PSlider" name="filterBrightParam">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="PSpinBox" name="spin_brightness">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="filterSwap">
     <property name="text">
      <string>swap</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PSlider" name="filterContrastParam">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="filterSwapH">
       <property name="text">
        <string>horizontally</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filterSwapV">
       <property name="text">
        <string>vertically</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="filterBrightContrast">
     <property name="text">
      <string>brightness:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_20">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>         contrast:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="PSpinBox" name="spin_contrast">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>filterBorder</zorder>
  <zorder>line_2</zorder>
  <zorder>filterBg</zorder>
  <zorder>filterBrightParam</zorder>
  <zorder>line</zorder>
  <zorder>filterSwap</zorder>
  <zorder>filterContrastParam</zorder>
  <zorder>label_20</zorder>
  <zorder>line_3</zorder>
  <zorder>filterBrightContrast</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PSpinBox</class>
   <extends>QSpinBox</extends>
   <header>pspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>PSlider</class>
   <extends>QSlider</extends>
   <header>pSlider.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterBrightContrast</tabstop>
  <tabstop>filterBrightParam</tabstop>
  <tabstop>spin_brightness</tabstop>
  <tabstop>filterContrastParam</tabstop>
  <tabstop>spin_contrast</tabstop>
  <tabstop>filterBorder</tabstop>
  <tabstop>filterBorderParamSize</tabstop>
  <tabstop>filterBorderParamSize_spin</tabstop>
  <tabstop>filterBorderParamCol</tabstop>
  <tabstop>filterBg</tabstop>
  <tabstop>filterBgShow</tabstop>
  <tabstop>filterBgUpdate</tabstop>
  <tabstop>filterBgReset</tabstop>
  <tabstop>filterBgLoad</tabstop>
  <tabstop>filterBgSave</tabstop>
  <tabstop>filterBgDeleteTrj</tabstop>
  <tabstop>filterBgDeleteNumber</tabstop>
  <tabstop>filterSwap</tabstop>
  <tabstop>filterSwapH</tabstop>
  <tabstop>filterSwapV</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>filterBorderParamSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>filterBorderParamSize_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterBorderParamSize_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>filterBorderParamSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterContrastParam</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_contrast</receiver>
   <slot>setValue(int)</slot>
  </connection>
  <connection>
   <receiver>filterContrastParam</receiver>
   <signal>valueChanged(int)</signal>
   <sender>spin_contrast</sender>
   <slot>setValue(int)</slot>
  </connection>
  <connection>
   <sender>filterBrightParam</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_brightness</receiver>
   <slot>setValue(int)</slot>
  </connection>
  <connection>
   <receiver>filterBrightParam</receiver>
   <signal>valueChanged(int)</signal>
   <sender>spin_brightness</sender>
   <slot>setValue(int)</slot>
  </connection>
 </connections>
</ui>
